TaskHero/ParseClient.swift

```diff
@@ -123,6 +123,66 @@ class ParseClient: NSObject {
         }
     }
     
+    func deleteTask(task: Task, success: @escaping () -> (), failure: @escaping (Error) -> ()) {
+        let query = PFQuery(className: "Task")
+        query.order(byAscending: "createdAt")
+
+        query.getObjectInBackground(withId: task.id!, block: { (object, error) -> Void in
+            if error == nil {
+                object!.deleteInBackground(block: { (deleteSuccess, error) in
+                    if error == nil {
+                        success()
+                    } else {
+                        failure(error!)
+                    }
+                })
+            } else {
+                failure(error!)
+            }
+        })
+    }
+    
+    func updateTask(task: Task, success: @escaping () -> (), failure: @escaping (Error) -> ()) {
+        let query = PFQuery(className: "Task")
+        query.order(byAscending: "createdAt")
```

Comment:
Again, just referencing the query.order(byAscending: "createdAt") line, but no biggie.